### ISSUE TITLE
DONOTCACHEPAGE

### ISSUE THREAD

**Issue by szepeviktor** [tlovett1/simple-cache#61]
Please support WooCommerce's DONOTCACHEPAGE contanst.

**Comment by tlovett1** [comment]
I'll investigate but this seems like a doable request.

**Comment by MartyD23** [comment]
Yeah, i was wondering something similar. Is it possible to exclude urls from cache?
